Packs/QRadar/Playbooks/playbook-QRadar_Indicator_Hunting_V2.yml

```diff
@@ -31,6 +30,11 @@ tasks:
     note: false
     timertriggers: []
     ignoreworker: false
+    continueonerrortype: ""
+    skipunavailable: false
```

Comment:
should it be true?

Reply:
This is the playbook start task, why it should be "True"? 
BTW, note that his is the radar playbook and not the threat hunting generic one